torch/csrc/jit/passes/onnx/fixup_onnx_controlflow.cpp

```diff
@@ -206,7 +206,9 @@ void ReplaceBlockOutputWithOptional(
   Node* opt_node = ONNXOptionalNode(opt_type, block->owningGraph());
```

Comment:
not this line, but the comment above for this function needs update. It states "Needed when control flow has multiple branches ..." which does not apply to "loop"

Reply:
As if and loop share this function, I keep the original description and add more comment of loop for this function.